src/rules/selector-no-id/index.js

```diff
@@ -19,6 +19,7 @@ export default function (actual) {
     root.walkRules(rule => {
       selectorParser(selectorAST => {
         selectorAST.eachId(id => {
+          if (/#{.+}/.test(id)) { return }
```

Comment:
> I guess the only time it would backfire is if somebody wanted to use a real id that like {something}

Do you mean CSS ID selectors? It should not be possible to have a name like that: https://stackoverflow.com/questions/448981/which-characters-are-valid-in-css-class-names-selectors/449000#449000